person.rb

```diff
@@ -23,6 +25,10 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
```

Comment:
- [ ] Great job. please ensure that The person itself should be sent as a parameter to create the new rental, achieving the 'has-many' association. [See more ](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)
something like this 
`def add_rental(book, date)`
   `Rental.new(date, book, self)`
`end`